amclient/amclientargs.py

```diff
@@ -132,6 +133,23 @@
     default="amclient-transfer",
     type=None,
 )
+JOB_MICROSERVICE = Opt(
+    name="job-microservice",
```

Comment:
I think for naming the parameters then this is the only one that doesn't feel as intuitive as the others. I feel like it should remain `microservice-group` if you agree as well. I agree that it is a grouping of Jobs, but from the UI'ss perspective, that grouping is the `microservice` or `microservice-group`.

Reply:
Interesting. Do you feel the the [parameter name of the API endpoint](https://github.com/artefactual/archivematica/pull/1440#issue-289156154) is also unintuitive?

```
* Parameters:
  * Optional filters
    * microservice: Name of the microservice the jobs belong to
```

Reply:
I think `microservice` and `microservice-group` are both pretty good as parameters. `microservice` sounds like the better option as it matches the API?

Reply:
Yes. I think we should keep it as `microservice` to make them match.

Reply:
To be clear though, not `job-microservice`? (sorry, I am being slow tonight!)

Reply:
Sorry I'm not being clear either :blush:. I meant `job-microservice` (since the other two Job related parameters in `amclient` are prefixed with `job-`).

Reply:
I see. Would `microservice-job` be okay? I understand the desire for consistency, but the first way around `job-microservice` reads to me `microservice that belongs to a job` and the second `microservice-job` reads `job that belongs to a microservice`. 

It might be too pedantic by me. I think the nicer option would be to drop the suffix because that's closer to the meaning in Archivematica. 

We can leave it for now too. I don't want to hold up this PR on semantics :wink:

Reply:
:+1: